src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function container-width($size, $columns, $sides-paddings) {
+  @return $size * $columns + $sides-paddings * 2;
+}
```

Comment:
you forgot to add gaps in your function; Look at the screen: https://prnt.sc/BKer0Wsht63s